app/broadcast_message/broadcast_message_schema.py

```diff
@@ -0,0 +1,48 @@
+from app.schema_validation.definitions import uuid
+from app.models import BroadcastStatusType
+
+create_broadcast_message_schema = {
+    '$schema': 'http://json-schema.org/draft-04/schema#',
+    'description': 'POST create broadcast_message schema',
+    'type': 'object',
+    'title': 'Create broadcast_message',
+    'properties': {
+        'template_id': uuid,
+        'service_id': uuid,
+        'created_by': uuid,
+        'personalisation': {'type': 'object'},
+        'starts_at': {'type': 'string', 'format': 'datetime'},
+        'finishes_at': {'type': 'string', 'format': 'datetime'},
+        'areas': {"type": "array", "items": {"type": "string"}},
```

Comment:
i thought that too - since we're moving away from building up a session and towards just saving in the DB and then updating, that makes it a lot simpler. i don't think there's really too much risk from this though